src/server.js

```diff
@@ -27,13 +27,13 @@ server.post('/', (request, response) => {
     const localDirectory = path.join(DEPLOY_DIR, name);
 
     try {
-      await setStatus('pending', `Staging at ${alias}`, alias);
+      await setStatus('pending', 'Staging...');
```

Comment:
Hmm, I'm torn. I like the idea of letting the user know ahead of time where the deploy will be. The goal is to have deterministic urls,  imo. But atm I don't think we have enough guarantees around the system to prevent giving back a url that could just 404.